ui/app/components/ui/popover/index.scss

```diff
@@ -0,0 +1,75 @@
+.popover {
+  &-wrap {
+    ::-webkit-scrollbar {
+      width: 6px;
+    }
+
+    ::-webkit-scrollbar-thumb {
+      -webkit-border-radius: 10px;
+      border-radius: 10px;
+      background: #C4C4C4;
+    }
+
+    position: absolute;
+    width: 328px;
+    height: 564px;
+
+    box-shadow: 0px 4px 30px rgba(0, 0, 0, 0.25);
+    border-radius: 10px;
+    background: white;
+  }
+
+  &-header {
+    &__heading {
+      @extend %font;
+      font-weight: bold;
+      font-size: 18px;
+      line-height: 25px;
+      text-align: center;
+      padding: 20px 0;
+      border-bottom: 1px solid #DDDEE9;
+    }
+
+    &__close {
+      position: absolute;
+      top: 13px;
+      right: 6px;
+      margin: 10px;
+      cursor: pointer;
+    }
+  }
+
+  &-bg {
+    width: 100%;
+    height: 100%;
+    background: black;
+    opacity: 20%;
+  }
+
+  &-content {
+    padding: 0 5px 0 16px;
+    margin-top: 5px;
+    margin-bottom: -10px;
+    margin-right: 5px;
+    height: calc(100% - 66px - 10px);
+    overflow-y: scroll;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: stretch;
+    align-content: stretch;
+  }
+
+  &-container {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 1050;
```

Comment:
Well this is unfortunate 😞 I'm guessing this was required to ensure this popup was shown on top? Do you know of any examples that forced this solution? I'd like to start purging z-index from the codebase, but we have to do that from the bottom-up, so I'm not blocking this from going in in the meantime of course.

Reply:
Right, there's a whole whack of `z-index in the .scss` so I just followed the existing ones that are uses for the current modal:

![image](https://user-images.githubusercontent.com/675259/77078056-74ad9800-69cc-11ea-87af-6801c8577b4a.png)

Dan and I are also thinking expunging `z-index` from the codebase is a worthwhile, but will probably be a lot of work